scrapy/spiders/__init__.py

```diff
@@ -78,6 +78,12 @@ def start_requests(self):
 
     def make_requests_from_url(self, url):
         """ This method is deprecated. """
+        warnings.warn(
+            "Spider.make_requests_from_url method is deprecated; it "
+            "won't be called by the default Spider.start_requests method "
```

Comment:
In addition to not being called, this method is also going to be removed. This is important if user code calls this method.

Reply:
oh indeed, thanks. I hope the updated text fits better.